### PR TITLE
Fix/nftpurchase url

### PR DESCRIPTION
**Pull request by aliirns**
## Description

Closes: #PS-226

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: big-dipper/imports/ui/easel_buy/EaselBuy.jsx [hunk 1]
```diff
@@ -222,6 +224,11 @@ export default class EaselBuy extends Component {
     window.location = `${baseURL}ofl=${ofl}&link=${encodeURIComponent(
       window.location.href
     )}`;
+
```
Comment (MikeSofaer) [inline big-dipper/imports/ui/easel_buy/EaselBuy.jsx:227]: What's this for?

File: big-dipper/imports/ui/easel_buy/EaselBuy.jsx [hunk 2]
```diff
@@ -451,6 +460,9 @@ export default class EaselBuy extends Component {
                                 <div className="item">
                                   <p>Creation Date</p>
                                   <p>
+                                    {console.log(createdAt,moment
```
Comment (MikeSofaer) [inline big-dipper/imports/ui/easel_buy/EaselBuy.jsx:463]: And this?

File: big-dipper/imports/api/recipes/server/methods.js [hunk 3]
```diff
@@ -56,7 +56,7 @@ Meteor.methods({
           let recipe = recipes[i];
           let deeplink =
             Meteor.settings.public.baseURL +
-            "?action=purchase_nft&recipe_id=" +
+            "?recipe_id=" +
```
Comment (mustafapylons) [inline big-dipper/imports/api/recipes/server/methods.js:59]: when we are building the deep link, do we need to sanitize the URL, any concerns?

**Review by mustafapylons — COMMENTED** [review]
one concern, otherwise code is good.

File: big-dipper/imports/ui/easel_buy/EaselBuy.jsx (hunk 1 above)
Comment (abdullahjankhan1) [reply big-dipper/imports/ui/easel_buy/EaselBuy.jsx:227]: used to generate deep link generation.

**Review by faddat — APPROVED** [review]

**Comment by faddat** [comment]
Is this ready to merge?  Still marked draft

**Review by faisalnaveedRNS — APPROVED** [review]

**Pull request merged**